### PR TITLE
Replace getRelationTypes with getValidRelationTypes…

### PR DESCRIPTION
**Pull request by audrey-denk**
… to prevent dups from showing

Change-Id: I0cac874d9ed1c25eaf13fd611afaf2cf8020d583

### CODE REVIEW

**Review by lucianovaglienti — APPROVED** [review]

File: plugins/org.eclipse.osee.framework.core/src/org/eclipse/osee/framework/core/data/OrcsTokenServiceImpl.java [hunk 1]
```diff
@@ -437,6 +437,40 @@ public Set<ArtifactTypeToken> getValidArtifactTypes(AttributeTypeToken attribute
       return artifactTypes;
    }
 
+   @Override
+   public List<RelationTypeToken> getValidRelationTypes() {
+      Collection<RelationTypeToken> relationTypes = getRelationTypes();
+      List<RelationTypeToken> invalidRels = new ArrayList<>();
+      //populate convertedRelations if empty
+      if (convertedRelations.isEmpty()) {
```
Comment (lucianovaglienti) [inline plugins/org.eclipse.osee.framework.core/src/org/eclipse/osee/framework/core/data/OrcsTokenServiceImpl.java:445]: Shouldn't this be done elsewhere?

File: plugins/org.eclipse.osee.framework.core/src/org/eclipse/osee/framework/core/data/OrcsTokenServiceImpl.java [hunk 2]
```diff
@@ -437,6 +437,40 @@ public Set<ArtifactTypeToken> getValidArtifactTypes(AttributeTypeToken attribute
...
+         setConvertedRelations();
+      }
+      //add all rels in shadowRels and remove the ones that have been converted
+      for (Field field : Arrays.asList(ShadowCoreRelationTypes.class.getDeclaredFields())) {
+         try {
+            RelationTypeToken rel = (RelationTypeToken) field.get(null);
```
Comment (lucianovaglienti) [inline plugins/org.eclipse.osee.framework.core/src/org/eclipse/osee/framework/core/data/OrcsTokenServiceImpl.java:451]: What is this doing?

File: plugins/org.eclipse.osee.framework.core/src/org/eclipse/osee/framework/core/data/OrcsTokenServiceImpl.java (hunk 1 above)
Comment (audrey-denk) [reply plugins/org.eclipse.osee.framework.core/src/org/eclipse/osee/framework/core/data/OrcsTokenServiceImpl.java:445]: Its done the first time this is called OR the first time a specific relation is called out.

File: plugins/org.eclipse.osee.framework.core/src/org/eclipse/osee/framework/core/data/OrcsTokenServiceImpl.java (hunk 2 above)
Comment (audrey-denk) [reply plugins/org.eclipse.osee.framework.core/src/org/eclipse/osee/framework/core/data/OrcsTokenServiceImpl.java:451]: This is for when we start actually converting relations.  Once DefaultHierarchy is converted for example, we would remove the "old" one from what can be returned as a valid relation.

**Review by ryanbase — APPROVED** [review]

**Pull request merged**